### ISSUE TITLE
Using lesser form to turn into a monkey while already being a monkey deletes you

### ISSUE THREAD

**Issue by soyconsumer** [vgstation-coders/vgstation13#19780]
1. Be Changeling
2. Use lesser form to turn into a monkey
3. Use lesser form again and choose monkey
4. Get deleted
5. Get kicked back into the game lobby
I was handcuffed if that's important

**Comment by kilozombie** [comment]
>get kicked back into the game lobby

That's wild as hell actually, you're sure you didn't just get nullspaced (everything turned black)?

**Comment by soyconsumer** [comment]
Well yes, everything turned black and I got disconnected but an admin said I got deleted and not nullspaced

**Comment by kilozombie** [comment]
If everything turns black (from what I know) it means you were deleted incorrectly/nullspaced.

**Comment by Exxion** [comment]
No, he was completely deleted. That or his mob's ckey got changed. That's how you end up back in the lobby.
But if you only got blackscreened and not disconnected, I guess you could have gotten nullspaced, then had the GC eat your mob while offline since it had no references to it

**Comment by SGuivenne** [comment]
I've just gotta say, this MIGHT need an exploitable tab by sheer virtue of allowing a psycho who roles changeling to murder a bunch of people or generally fuck the shit out of everything, nullify themselves from existence, and then just rejoin again and theoretically be immune to any consequences as you are no longer an antag and technically no longer the same person.
